sdk/trustedsigning/azure-mgmt-trustedsigning/apiview-properties.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
@msyyc is this apiview json file expected to be included in the SDK PR?

Reply:
Yes.